src/App.js

```diff
@@ -1,29 +0,0 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
```

Comment:
As of the project requirement and good practice of react projects you have to have `App.js` file where you will import you components and render out there then the `App.js` component it self will be render in the `index.js` file. Here is how you have to have your `App.js` component where the `Calculator.js` component will be rendered out.

```
import React, { Component } from 'react';
import Calculator from your calculator component path

class App extends Component {
  render() {
    return {
      <Calculator />
    };
  }
}

```